### ISSUE TITLE
Soft hyphen (00AD) is invisible

### ISSUE THREAD

**Issue by drs-p** [CatharsisFonts/Cormorant#80]
The soft hyphen in the Cormorant fonts is invisible and has width zero. That is a problem in a Tagged PDF as soft hyphens should be used for hyphenated words to improve accessibility and screen reading. See the discussion at this issue: https://github.com/latex3/tagging-project/issues/967, and also https://jkorpela.fi/shy.html#pdf for a discussion about the soft hyphen in general and its use in PDF specifically.
